### PR TITLE
Move some inline functions from libpe_status to libcrmcommon

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this when you get a chance? Low priority

File: include/crm/common/clone_internal.h [hunk 1]
```diff
@@ -59,6 +59,20 @@ pcmk__is_unique_clone(const pcmk_resource_t *rsc)
...
+/*!
+ * \internal
+ * \brief Check whether a resource is an anonymous clone
+ *
+ * \param[in] rsc  Resource to check
+ *
+ * \return true if \p rsc is an anonymous clone, otherwise false
+ */
+static inline bool
+pcmk__is_anonymous_clone(const pcmk_resource_t *rsc)
```
Comment (nrwahl2) [inline include/crm/common/clone_internal.h:71]: Should be a refactor commit, not an API commit

File: include/crm/pengine/status.h [hunk 2]
```diff
@@ -80,6 +67,10 @@ pe_rsc_is_bundled(const pcmk_resource_t *rsc)
     return rsc->variant == pcmk_rsc_variant_bundle;
 }
 
+#if !defined(PCMK_ALLOW_DEPRECATED) || (PCMK_ALLOW_DEPRECATED == 1)
```
Comment (nrwahl2) [inline include/crm/pengine/status.h:51]: Seems like this should be in an earlier commit but doesn't matter

File: include/crm/common/bundles_internal.h [hunk 3]
```diff
@@ -27,6 +27,26 @@ typedef struct {
...
+ *
+ * \return true if \p rsc is part of a bundle, otherwise false
+ */
+static inline bool
+pcmk__is_bundled(const pcmk_resource_t *rsc)
+{
+    if (rsc == NULL) {
+        return false;
+    }
+    while (rsc->parent != NULL) {
```
Comment (nrwahl2) [inline include/crm/common/bundles_internal.h:44]: I know this is copy-paste, but we could probably use `pe__const_top_resource` here.

**Review by nrwahl2 — COMMENTED** [review]
Looks good

File: include/crm/common/clone_internal.h (hunk 1 above)
Comment (kgaillot) [reply include/crm/common/clone_internal.h:71]: Whoops, updated

File: include/crm/pengine/status.h (hunk 2 above)
Comment (kgaillot) [reply include/crm/pengine/status.h:51]: Ah yes, I shuffled commits around

**Comment by kgaillot** [comment]
Updated per review

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**